### DISCUSSION TITLE
Can there be a version of Figma design file which can be used without requiring Enterprise subscription for publishing on Figma?

### DISCUSSION THREAD

**Discussion by aDisplayName** [microsoft/fluentui#32240] [Ideas]
The current version of figma design file requires Enterprise subscription to be used as library, potentially caused by more then 4 variable modes:

![image](https://github.com/user-attachments/assets/ceca3b45-c43d-4164-b12a-52f608aabcee)

Is there a plan to release a lite version of the design file for folks with lower tier to use?

**Comment by darylohrt** [comment]
If you publish everything but the avatar color variables collection, it works.    You will still not have the nav and the avatar colors.    I think this is fixable locally, but I haven't needed to try yet.   Should be quick fix but given this is not up to date with the lastest version i dont know who useful it will be.
